onnxruntime/core/optimizer/reshape_fusion.cc

```diff
@@ -57,20 +57,15 @@ DistilBert reshape pattern:
                 \      /
                 Reshape
 */
-static bool DistilBert_Check(Graph& graph, const Node& concat, const Node& layernorm_add, const logging::Logger& logger) {
+static bool DistilBert_Check(Graph& graph, const Node& concat, const Node& root, const logging::Logger& logger) {
   std::vector<graph_utils::EdgeEndToMatch> shape_path{
     {0, 0, "Unsqueeze", {1, 11}, kOnnxDomain},
     {0, 0, "Gather", {1, 11}, kOnnxDomain},
-    {0, 0, "Shape", {1}, kOnnxDomain},
-    {0, 0, "Add", {7, 13}, kOnnxDomain}};
+    {0, 0, "Shape", {1}, kOnnxDomain}};
   std::vector<const Node::EdgeEnd*> edges;
   if (!graph_utils::FindPath(concat, true, shape_path, edges, logger)) {
```

Comment:
Such path has been matched in Match_One_Element_Output_Subgraph_1. Need we do it again?